ui_framework/components/modal/modal.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export function KuiModal({ children, ...rest }) {
+  return (
+    <div className="kuiModal" { ...rest }>
```

Comment:
Pulled out className to avoid overwriting them.

@kjbekkelund... so you are saying use `...rest` for this component because it's a simple wrapper, but for the likes of KuiConfirmModal, which is more complicated, manually pass them in?

I'm fine with whatever.

What I think would be nice is a helper function to build these components, where all they are doing is using a className with children.  There are so many of them that look identical except for a class name.  But I hear a lot of feedback that I tend to over-DRY so I'll leave that alone for now. :)